app/views/posts/show.html.erb

```diff
@@ -1,6 +1,14 @@
-<h1>Sample Post</h1>
-<p>Sample Comment<p>
-<p>Sample Comment<p>
-<p>Sample Comment<p>
-<p>Sample Comment<p>
-<p>Sample Comment<p>
\ No newline at end of file
+<div class="shadow-white rounded border p-3 m-5">
+  <div class="d-flex justify-content-between mb-4">
+    <h4><%= @post.title %> by <%= @user.name %></h4>
+    <h5>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></h5>
```

Comment:
- [ ] **[OPTIONAL]** The likes and comments counters for a post are also not being updated. I have a comment, but the counter still reads 0.

  ![image](https://github.com/matee-safi/rails-blog/assets/36542986/c078b4f4-13ca-43b9-a8f1-f2ac8fae3af5)